cfpb/cfgov-refresh
# YES tool — Add view for `daysPerWeek` fields

Introduces a view to manage the combo field inputs for `daysPerWeek` data. This property is the number of times per week a user expects to take the chosen type of transportation.

## Additions

- `DaysPerWeekView`
- New reducer actions to clear `daysPerWeek` associated data


## Removals

- Some specs / code that was not applicable any more

## Changes

- Adds view as a child of `RouteOptionView`

## Testing

1. On the transportation tool:
* Fill in the budget fields 
* Select the transportation option `Drive`
* Fill in the `milesPerDay` and `daysPerWeek` fields.
* Observe values are calculated
* Select the `not sure` checkbox and observe that the todo list has
an item associated with it
* Deselect for the reverse behavior
* With both fields filled in, select a transportation option other than drive
* Reselect drive, fields should be cleared (and todo list empty)

## Screenshots


## Notes

- Depends on #5227 

## Todos

- View specs
- JSDoc

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
